examples/xdc-example/src/app/xdc.subscriptions.example.ts

```diff
@@ -9,13 +9,14 @@ export async function xdcSubscriptionsExample() {
     type: 'ACCOUNT_INCOMING_BLOCKCHAIN_TRANSACTION',
     attr: {
       id: '635b92d6f9aeb823db30c43f',
-      url: 'dashboard.tatum.io/webhook-handler',
+      url: 'https://dashboard.tatum.io/webhook-handler',
```

Comment:
if we do account incoming webhook - account must be initialized here before. not random accountID provided